Life.java

```diff
@@ -42,11 +44,69 @@ public void setDead(int x, int y) {
   }
 
   public boolean isAlive(int x, int y) {
-    return raster[x][y];
+  // Verhindern von null Pointer Exception
```

Comment:
so was macht man gerne, alles was außerhalb ist ist sozusagen tot.
Das führt aber oft zu versteckten Fehlern, die man gar nicht mehr so leicht findet.
Eigentlich sollte das doch gar nicht vorkommen: Ein ungültiger Index. Das ist Fehler!
Wie behandeln wir solche Fehler in Java? Wir werfen eine Exception! Hier z.B. eine IndexOutOfBounds Exception! Glücklicherweise macht das Java schon für uns.